PartTwo/PartTwo/AVColorPickerCommProtocol.h

```diff
@@ -0,0 +1,15 @@
+//
+//  AVColorPickerCommProtocol.h
+//  PartTwo
+//
+//  Created by Artur Lan on 9/18/15.
+//  Copyright © 2015 Mike Kavouras. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@protocol AVColorPickerCommProtocol <NSObject>
+
+-(void)didSelectColor:(UIColor *)color;
```

Comment:
Consider making this more general. `didTapCommentButton` instead of `didSelectColor`. `didSelectColor` implies intent, and the protocol should be oblivious to intent.